src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_RankSumTest.java

```diff
@@ -0,0 +1,121 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
```

Comment:
We should probably move all of the allele-specific annotations into their own subpackage within `annotations`